### PR TITLE
Separated the NetModule to dependant and standalone

### PR DESCRIPTION
**Pull request by pokk**
as the title.

### CODE REVIEW

**Comment by pokk** [comment]
@lekaha 

Please take a look. 🙏

File: mobile-ui/src/standalone/java/com/playone/mobile/ui/firebase/StandAloneAuthenticator.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package com.playone.mobile.ui.firebase
+
+import com.google.firebase.auth.FirebaseUser
+import com.playone.mobile.domain.Authenticator
+import com.playone.mobile.domain.Credential
+import com.playone.mobile.domain.model.User
+
+class StandAloneAuthenticator : Authenticator() {
+    val mapper = FirebaseUserMapper()
```
Comment (lekaha) [inline mobile-ui/src/standalone/java/com/playone/mobile/ui/firebase/StandAloneAuthenticator.kt]: empty line please

File: mobile-ui/src/standalone/java/com/playone/mobile/ui/firebase/StandAloneAuthenticator.kt [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package com.playone.mobile.ui.firebase
+
+import com.google.firebase.auth.FirebaseUser
+import com.playone.mobile.domain.Authenticator
+import com.playone.mobile.domain.Credential
+import com.playone.mobile.domain.model.User
+
+class StandAloneAuthenticator : Authenticator() {
```
Comment (lekaha) [inline mobile-ui/src/standalone/java/com/playone/mobile/ui/firebase/StandAloneAuthenticator.kt]: `Standalone` is a word so I think it is better do not use `StandAlone`

File: mobile-ui/src/standalone/java/com/playone/mobile/ui/firebase/StandAloneAuthenticator.kt [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    override fun signOut(callback: AuthResultCallBack) = TODO()
+
+    override fun isSignedIn() = false
+
+    override fun isVerifiedEmail() = TODO()
+
+    class FirebaseUserMapper {
+
+        fun mapToUser(user: FirebaseUser) =
+            User().apply {
```
Comment (lekaha) [inline mobile-ui/src/standalone/java/com/playone/mobile/ui/firebase/StandAloneAuthenticator.kt]: empty line!

File: mobile-ui/src/standalone/java/com/playone/mobile/ui/firebase/StandAloneAuthenticator.kt [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+    override fun signInAnonymously(callback: AuthResultCallBack) = TODO()
+
+    override fun signOut(callback: AuthResultCallBack) = TODO()
+
+    override fun isSignedIn() = false
+
+    override fun isVerifiedEmail() = TODO()
+
+    class FirebaseUserMapper {
```
Comment (lekaha) [inline mobile-ui/src/standalone/java/com/playone/mobile/ui/firebase/StandAloneAuthenticator.kt]: I think this Mapper is not needed for Standalone

**Comment by pokk** [comment]
@lekaha 

I've fixed! Please take a look again. 🙏

**Review by lekaha — APPROVED** [review]

**Pull request merged**